### ISSUE TITLE
Add ability to sum collected rider fares in runs.

### ISSUE THREAD

**Issue by keviniano** [full-path/ridesheet#70]
If collected fares are tracked at a per-rider level, RideSheet should be able to generate a sum of those fares and track them at the run level.

**Comment by NomeQ** [comment]
@keviniano I suspect it will be a long time before we decide to integrate fare capabilities into `main`, so I think we can close this issue.

**Comment by keviniano** [comment]
Agreed. I have a version of this for Clay and Marshall Counties through a calculated field, but it's not needed in main.
